JapaneseFilter/sections/specific.txt

```diff
@@ -99,6 +99,15 @@ weblio.jp##.adBannerHdWrp
 weblio.jp##.adSmpBnrFtBtSp
 weblio.jp##.adBkOverlayClr
 new-akiba.com###bannerB2
+iza.ne.jp##.header-link > ul > li > a[href="http://www.sankei-digital.co.jp/"]
+||prt.iza.ne.jp/common/css/v2/images/bg_header_003.png^$domain=iza.ne.jp
+||prt.iza.ne.jp/common/images/v1/basic/banner_sankeiPhoto.png^$domain=iza.ne.jp
+iza.ne.jp##article.article-detail-section > .tool > .credit
+iza.ne.jp##.sideFixedWrapper > div[style="width:300px;height:600px;"]
+iza.ne.jp##section.module-pickup-topics
+iza.ne.jp##aside.ad-special
+iza.ne.jp##aside.ad-trend
+iza.ne.jp##aside.ad-tokudane
```

Comment:
All those aside.ad rules can be done with this rule:
iza.ne.jp##aside[class^="ad-"]